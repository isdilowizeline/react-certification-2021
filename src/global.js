import {createGlobalStyle} from 'styled-components';

export const GlobalStyles = createGlobalStyle`
  html, body {
    margin: 0;
    padding: 0;
    font-size: 1.125rem;
    line-height: 1.6;
    font-weight: 400;
    font-family: sans-serif;
    box-sizing: border-box;
    scroll-behavior: smooth;
    -webkit-tap-highlight-color: rgba(0, 0, 0, 0);
  }
  *, *::after, *::before {
    box-sizing: inherit;
  }
  body {
    background: ${({theme}) => theme.primaryBackground};
    color: ${({theme}) => theme.primaryHover};
    text-rendering: optimizeLegibility;
    font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
    text-rendering: optimizeLegibility;
  }
  h1 {
    font-size: 2rem;
    text-align: center;
    text-transform: uppercase;
  }
  .separator::before {
    content: 'â€¢';
    color: white;
    padding: 0.4rem;
  }
   
  hr {
  }
  
  .burgerDiv {
    text-align: center;
    box-sizing: border-box;
  }
  
  small {
    display: block;
  }
  
  a {
    color: ${({theme}) => theme.primaryHover};
    font-weight: bold;
    text-decoration: none;
  }
  
  a:active {
    color: blueviolet;
  }
  
.header {
    color: #fff;
    height:70px;
    background: ${({theme}) => theme.primaryDark};
}
.headerRow {
padding-top:10px;
}
`;
