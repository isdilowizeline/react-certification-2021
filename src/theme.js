export const theme = {
    primaryDark: '#1C5476',
    primaryBackground: '#FFFFFF',
    primaryLight: '#FFFFFF',
    primaryLight2: '#000',
    primaryHover: '#343078',
    mobile: '576px',
}