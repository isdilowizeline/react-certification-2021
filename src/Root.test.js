import React from 'react';
import {render} from '@testing-library/react';
import {GlobalStyles} from './global';
import {ThemeProvider} from "styled-components";
import {theme} from './theme';

describe('Root File Test', () => {
    test('should take a snapshot', () => {
        const {asFragment} = render(
            <ThemeProvider theme={theme}>
                <GlobalStyles/>
            </ThemeProvider>
        );
        expect(
            asFragment(
                <ThemeProvider theme={theme}>
                    <GlobalStyles/>
                </ThemeProvider>
            )
        ).toMatchSnapshot();
    });
});