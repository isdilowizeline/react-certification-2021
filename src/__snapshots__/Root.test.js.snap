// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Root File Test should take a snapshot 1`] = `<DocumentFragment />`;
